paddle/fluid/framework/new_executor/standalone_executor_test.cc

```diff
@@ -145,23 +145,79 @@ TEST(StandaloneExecutor, run) {
   exec.Run({}, {}, {});
   auto start = std::chrono::steady_clock::now();
 
-  // ProfilerStart("new_executor.prof");
-
-  for (size_t i = 0; i < 2320; ++i) {
+  for (size_t i = 0; i < 10; ++i) {
```

Comment:
why change this?

Reply:
No need to run 2320 rounds since it now only tests whether the standalone executor can be successfully executed.